docs/Tomcat_Operation_Flow.md

```
@@ -0,0 +1,52 @@
+
+- ServerSocket의 동작 방식
+    - serverSocket.accept()
+        - 현재 연결 요청이 있으면 연결 객체인 Socket을 생성해서 리턴
+        - 현재 연결 요청이 없으면 연결이 도착할 때까지 대기
+
+### Socket
+
+- Socket
+    - 클라이언트와의 실제데이터 통신을 수행하는 연결 객체
+- Socket의 동작 방식
+    - socket.getInputStream()
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)